_layouts/plugins/katello/documentation.html

```diff
@@ -1,6 +1,6 @@
 ---
 pagename: Plugin Manuals
-versions: [{name: 3.8, display: '3.8'}, {name: 3.7, display: '3.7'}, {name: 3.6, display: '3.6'}, {name: 3.5, display: '3.5'}, {name: 3.4, display: '3.4'}, {name: 3.3, display: '3.3'}, {name: 3.2, 'display': '3.2'}, {name: 3.1, display: '3.1'}, {name: 3.0, display: '3.0'}, {name: 2.4, display: '2.4'}, {name: 'nightly', display: 'nightly'}]
+versions: [{name: 3.9, display: '3.9'}, {name: 3.8, display: '3.8'}, {name: 3.7, display: '3.7'}, {name: 3.6, display: '3.6'}, {name: 3.5, display: '3.5'}, {name: 3.4, display: '3.4'}, {name: 3.3, display: '3.3'}, {name: 3.2, 'display': '3.2'}, {name: 3.1, display: '3.1'}, {name: 3.0, display: '3.0'}, {name: 2.4, display: '2.4'}, {name: 'nightly', display: 'nightly'}]
```

Comment:
You want to set the display here to `3.9-RC` so users dont think its been released yet.

Reply:
Updated, thanks!